src/test/java/test/NaptolSortByAddedProductToCart.java

```diff
@@ -0,0 +1,44 @@
+package test;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import pojo.Browser;
+import pom.NaptolCartPage;
+import pom.NaptolHomePage;
+import pom.NaptolProductSpecificationPage;
+import pom.ProductSearchPage;
+
+public class NaptolSortByAddedProductToCart extends BaseTest{
```

Comment:
Class name should be short and more meaningful.

Reply:
Done.